From 75b5bc6c1a689a97736f6088b5fa3e48432ef3cf Mon Sep 17 00:00:00 2001
From: Mike Schuchardt <mikes@lunarg.com>
Date: Wed, 15 Jan 2020 16:38:27 -0800
Subject: [PATCH] Fix GN build

Vulkan-Headers uses a new build_override include to toggle x11 support
---
 .../secondary/build_overrides/vulkan_headers.gni  | 15 +++++++++++++++
 1 file changed, 15 insertions(+)
 create mode 100644 build-gn/secondary/build_overrides/vulkan_headers.gni

diff --git a/build-gn/secondary/build_overrides/vulkan_headers.gni b/build-gn/secondary/build_overrides/vulkan_headers.gni
new file mode 100644
index 000000000..0cd8307dc
--- /dev/null
+++ b/build-gn/secondary/build_overrides/vulkan_headers.gni
@@ -0,0 +1,15 @@
+# Copyright (c) 2020 LunarG, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+vulkan_use_x11 = true
